gammapy/estimators/points/core.py

```diff
@@ -322,6 +322,14 @@ def to_table(self, sed_type="likelihood", format="gadf-sed", formatted=False):
 
             idx = (Ellipsis, 0, 0)
             table = self.energy_axis.to_table(format="gadf-sed")
+
+            if self.energy_axis.nbin == 1 and self.energy_axis.node_type == "center":
+                if sed_type in ["flux", "eflux", "likelihood"]:
```

Comment:
So this clause is because Gammapy has no way of guessing the energy edges given a single centre?

But I suppose we are still guessing the edges given multiple centres? and, say, for flux points, it can be totally incorrect for the last bin?

Reply:
Thanks, I now realize this is not a complete solution and probably requires a some more thought. The problem is basically that when computing flux points in Gammapy we know `e_min`, `e_max` and `e_ref`. For completeness we should add this information to the table for the `dnde` case as well. However in the case where we read existing flux points of type `dnde` we should indeed not invent the information. I need to think bit more how to better handle this maybe we need an additional keyword on the `MapAxis` object...